src/arm/midr.c

```diff
@@ -21,6 +25,66 @@
 #include "uarch.h"
 #include "sve.h"
 
+
+#if defined _WIN32
+// Windows stores processor information in registery at:
+// "HKEY_LOCAL_MACHINE\HARDWARE\DESCRIPTION\System\CentralProcessor"
+// Within this directory, there each core will get its own folder as well with
+// registery entries that map to system registers on ARM in the `CP ####` registry entries
+// Ex. the MIDR register for core 0 is the `REG_QWORD` at:
+// "HKEY_LOCAL_MACHINE\HARDWARE\DESCRIPTION\System\CentralProcessor\0\CP 4000"
+// The name of these `CP ####`-registers follow their register ID encoding in hexadecimal
+// (op0&1):op1:crn:crm:op2.
+// More registers can be found here:
+// https://developer.arm.com/documentation/ddi0601/2024-06/AArch64-Registers
+// Some important ones:
+// CP 4000: MIDR_EL1
```

Comment:
It's cool you can read those without root/admin permissions...definitely would be nice to have this in Linux as well!